boefjes/boefjes/plugins/kat_external_db/normalize.py

```diff
@@ -29,6 +31,18 @@ def follow_path_in_dict(path, path_dict):
     return path_dict
 
 
+def get_indemnification_level(path_dict, path=INDEMNIFICATION_ITEM_PATH, default=DEFAULT_INDEMNIFICATION_LEVEL):
```

Comment:
I'm not sure using tuples as default values is a good idea here.

Reply:
The code also doesn't use these arguments, so the arguments should be removed and the default can be used directory in the code.

Reply:
I can change it, although using variables was per-sorting for when normalizer settings are available.

Reply:
This way it is easier to change the settings for a different DB because they are grouped together in the file.

Reply:
I am fine with the constants defined in the top of the file.

What I meant we can just use the constants in the code instead of defining function arguments that are never used by callers. Adding function arguments implies that the function is called with those arguments, making it in my opinion confusing because that doesn't happen. It's just a small thing here and on itself it is not that really a big problem, but what I am concerned with is that if we do this everywhere we get a more complex and harder to understand codebase in general.

To prevent this I am fan of the YAGNI (You Aren't Gonna Need It) principle: only create the code that is needed now and create the code that is needed later when it needed. In my opinion the principle applies both to whole features and smaller code parts such as here.